team-policy.md

```diff
@@ -0,0 +1,144 @@
+
+The agenda backlog and minutes document is structured according to the following template:
+
+```md
+# T-style Minutes
+
+Meeting Link: https://meet.jit.si/t-style-style-team-task-force
+
+## Action Items
+
+### Pending
+
+* owner: bullet point list of items that are in progress and assigned to a specific person
+
+### Completed
+
+- [ ] owner: check list of items that are completed or assumed to be complete
+- [x] owner: items are checked off once they've been reviewed in a meeting, confirmed complete, and given any relevant final status updates.
+- [x] owner: after a meeting the items that were checked off are moved into the `#### Completed Action Items` section of the meeting they were reviewed in by the secretary
+
+## Backlog
+
+* Bullet point list of items that have not been started or assigned yet
+
+## Next Meeting Date
+
+### Attendance
+
+### Agenda
+
+* (inform) bullet point list of proposed agenda items (labeled either inform, explore, or decide)
+* Review Action Items
+* Meeting Check-out
+    * Do not record in minutes, exceptions can be made with consent of team
+
+### Minutes
+
+#### Individual Agenda Items
+
+Notes related to the given agenda item
+
+#### Completed Action Items
+
+## Previous Meeting Dates <repeats>
+
+```
+### Meeting Check-out
+
+The style team wraps up the content part of our meetings 5-10 minutes before team members have to leave to make room for regular meeting evaluations. Meeting evaluations are an integral part of every meeting. We end the meeting with one or two rounds on:
+
+* “What worked well in the meeting?”
+* “What could be improved in future meetings?”
+* “Is there anything you are carrying out of the meeting that you’d like to get off your chest now?”
+
+Meeting evaluations are an opportunity to learn from our meetings. We can either talk about content, process, or interpersonal dynamics. We utilize meeting evaluations to help ourselves inhabit a growth mindset. Our goal is to have meetings which are well-run, refreshing, connecting, and energizing. We achieve this goal by giving space for people to speak up about both the positive and negative aspects of how we're working together so that we can prioritize continuous improvement and positive connections.
+
+Meeting check-outs are considered private and internal to the style team and are not recorded as part of our minutes. Exceptions to this can be made via an operational consent decision by the team, and are often useful in cases such as when new backlog or action items come up during the check-out.
+
+[Style Guide]: https://github.com/rust-lang/fmt-rfcs/blob/master/guide/guide.md
+[#T-style]: https://rust-lang.zulipchat.com/#narrow/stream/346005-t-style
```

Comment:
This appears to be referenced in only one place, and I don't think that place needs it to be a link. Can we drop this?

```suggestion
```